File: app/src/main/java/com/jajinba/pixabaydemo/utils/ArrayUtils.java
package com.jajinba.pixabaydemo.utils;

import java.util.Collection;

public class ArrayUtils {

  public static int getLengthSafe(Collection<?> collection) {
    return collection == null ? 0 : collection.size();
  }

  public static boolean isEmpty(Collection<?> collection) {
    return getLengthSafe(collection) == 0;
  }

  public static boolean isNotEmpty(Collection<?> collection) {
    return getLengthSafe(collection) > 0;
  }
}

File: app/src/main/java/com/jajinba/pixabaydemo/utils/ArrayUtils.kt
package com.jajinba.pixabaydemo.utils

object ArrayUtils {
    fun getLengthSafe(collection: MutableCollection<*>?): Int {
        return collection?.size ?: 0
    }

    fun isEmpty(collection: MutableCollection<*>?): Boolean {
        return getLengthSafe(collection) == 0
    }

    fun isNotEmpty(collection: MutableCollection<*>?): Boolean {
        return getLengthSafe(collection) > 0
    }
}